packages/apps/docs/src/docs/build/index.tsx

```diff
@@ -43,13 +43,9 @@ const Home: FC<IProps> = ({ blogPosts, popularPages }) => {
                 clicking Quick start.
               </Text>
             </Box>
-            <Link
-              endIcon={<SystemIcon.TrailingIcon />}
-              variant="contained"
-              href="/build/quickstart"
-            >
+            <Button icon={<SystemIcon.TrailingIcon />} href="/build/quickstart">
```

Comment:
This for me really feels counter-intuitive as the button is now a Link and I can't read from the tag name what the purpose of the element is. what was the reason for changing this?

Reply:
Personal experience in previous libraries and talks with other developers. Some prefer a button which can handle both and some don't. 

I'm trying to get feedback on this but I'm already thinking about splitting them again due to recent insights.